File: 0125-valid-palindrome/0125-valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
       String s1 = s.replaceAll("[^a-zA-Z0-9]","");
       StringBuffer s2 = new StringBuffer(s1);
       if(s1.equalsIgnoreCase(s2.reverse().toString())){
         return true;
       }else{
        return false;
       }
        
    }
}

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string ans = "";
        for(int i=0;i<s.length();i++){
            if(isalnum(s[i])){
                ans+= tolower(s[i]);
            }
        }
        
        int l = 0;
        int r = ans.length()-1;
        while(l < r){
            if(ans[l]!=ans[r])
                return false;
            else
                l++ , r--;
        }
        return true;;
    }
};